platform/source/lib/ax/platform/rest.py

```diff
@@ -228,7 +227,14 @@ def put_spot_instance_config():
 @_app.route("/v1/axmon/cluster/spot_instance_config", methods=['GET'])
 def get_spot_instance_config():
     response = requests.get(MINION_MANAGER_HOSTNAME + ":" + MINION_MANAGER_PORT + "/spot_instance_config")
-    return response.text
+    details = response.json()
+    assert "asgs" in details, "No asgs returned by minion-manager"
+
+    spotOptionMgr = SpotInstanceOptionManager(
+        os.getenv("AX_CLUSTER_NAME_ID"), AXClusterConfig().get_region())
+    spot_option = spotOptionMgr.asgs_to_option(details["asgs"].split(" "))
+
+    return jsonify({"status": details["status"], "spot_instances_option": spot_option})
```

Comment:
Usually, when your HTTP PUT data has fields "enabled" and "spot_instances_option", you should include them in response as well. Also, pls document what a response looks like so people in the community knows how to use these APIs

Reply:
Will include this in the README.